pemutil/pem.go

```diff
@@ -740,3 +740,38 @@ func BundleCertificate(bundlePEM []byte, certsPEM ...[]byte) ([]byte, bool, erro
 
 	return bundlePEM, modified, nil
 }
+
+// UnbundleCertificate removes PEM-encoded certificates from a PEM-encoded
+// certificate bundle.
+func UnbundleCertificate(bundlePEM []byte, certsPEM ...[]byte) ([]byte, bool, error) {
+	drop := make(map[[sha256.Size224]byte]bool, len(certsPEM))
+	for i := range certsPEM {
+		cert, err := ParseCertificate(certsPEM[i])
```

Comment:
I can see using this function but having `certsPEM[i]` as a certificate bundle. If a read a bundle and want to remove one cert, I don't want to split the bundle in multiple certs.

Reply:
9286a0c